frontend/src/modules/displays/ToggleValue.tsx

```diff
@@ -45,6 +49,16 @@ const StyledToggleButton = withStyles(({ palette }) => ({
   },
 }))(ToggleButton);
 
+/**
+ * @typedef Props
+ *
+ * Props Interface for the ToggleValue component
+ *
+ * @prop {string} toggleBetween.label Toggle Input label
```

Comment:
Can you give a few examples of the kinds of things we would provide as the label of a toggle input?

Reply:
Toggle label example in our app would be theme color setting. Here input field toggles between Dark & Light theme. So the label would be `Dark UI` & `Light UI`